lib/std/enumutils.nim

```diff
@@ -36,6 +36,10 @@ macro genEnumCaseStmt*(typ: typedesc, argSym: typed, default: typed,
     case f.kind
     of nnkEmpty: continue # skip first node of `enumTy`
     of nnkSym, nnkIdent: fStr = f.strVal
+    of nnkAccQuoted:
```

Comment:
since you're doing this, please address the above comment:
> # xxx `genEnumCaseStmt` needs tests and runnableExamples

Reply:
You as the main author (afaik) of this submodule are much better suited to writing those tests. You know the use cases etc. much better than I do.

I would have already written a test case *for the issue I'm having* if it didn't just happen under `nim doc` (which I have no idea how to test).

Reply:
> You as the main author (afaik) of this submodule are much better suited to writing those tests. You know the use cases etc. much better than I do.

i wrote the rest of the module, but not `genEnumCaseStmt`; we really need to have a runnableExamples+tests mentality.

> I would have already written a test case for the issue I'm having if it didn't just happen under nim doc (which I have no idea how to test).

see `trunner` which has lots of examples involving custom testing scenarios (eg for nim doc), look how it's done there

Reply:
> we really need to have a runnableExamples+tests mentality.

You don't have to tell that to me, I fully agree. 

> see trunner which has lots of examples involving custom testing scenarios (eg for nim doc), look how it's done there

I'll take a look, but it will probably be a while until I have the time to dig into that + write tests.